app/uk/gov/hmrc/apigatekeeperapprovalsfrontend/controllers/TermsOfUseHistoryController.scala

```diff
@@ -43,7 +43,8 @@ object TermsOfUseHistoryController {
       applicationName: String,
       historyEntries: List[TermsOfUseHistory],
       applicationDetailsUrl: String,
-      isInHouseSoftware: Boolean
+      isInHouseSoftware: Boolean,
+      dueDate: String
```

Comment:
Does this want to be a string or a Date ?

Reply:
I was just going to add it to the view model as a string and do the formatting in the controller

Reply:
If you like.  I just thought it was unusual even for a view model - but not unacceptable.